src/scss/_sofie-table.scss

```diff
@@ -0,0 +1,70 @@
+.c-sofie-table {
```

Comment:
Because I'm not quite sure how to create a generic table component without saying that the data this component will receive will be array<any>. Until now, when I have built such components, I give the table keys and the order of the columns as an object set from the parent component. Similar to the logic I have in sofie-table-header. However, this cannot be done without providing a table-data sheet which must be an array<any> as far as the sofie-table component is concerned.